### PR TITLE
Update PGP on Android (Remove APG, Update OpenKeychain)

### PR DESCRIPTION
**Pull request by dschuermann**
* Updates OpenKeychain logo
* Use https for OpenKeychain homepage
* removes APG, see https://github.com/nylira/prism-break/pull/1171, https://github.com/nylira/prism-break/issues/1156
* Remove mentions of APG in ``email-encryption-notes`` for en and de

### CODE REVIEW

**Comment by dschuermann** [comment]
Also: https://k9mail.github.io/2017/01/13/Why-APG-is-no-longer-supported.html

File: source/locales/de.json [hunk 1]
```diff
@@ -133,7 +133,7 @@
...
... „Ziemlich gute Privatsphäre“) ist ein Programm zur Verschlüsselung und zum Unterschreiben von Daten.&rdquo;</p><p>&mdash; <a href=\"https://de.wikipedia.org/wiki/Pretty_Good_Privacy\">Wikipedia</a><br><p>PGP wird häufig zum signieren, verschlüsseln und entschlüsseln von Texten, E-Mails, Dateien, Ordnern und ganzen Festplattenpartitionen genutzt, um die Sicherheit von E-Mailkommunikation zu erhöhen.</p><footer>&mdash; <cite><a href='https://de.wikipedia.org/wiki/Pretty_Good_Privacy'>Wikipedia</a></cite></footer></blockquote><p>PRISM Break empfiehlt die S/MIME E-Mail Verschlüsselung nicht, da es Drittanbieter-Zertifikate von zentralen Autoritäten benötigt. Mehr <a href='https://www.enigmail.net/forum/viewtopic.php?f=7&amp;t=67'>hier (Englisch)</a>.</p><p><strong>OpenKeychain</strong> zusammen mit <strong>K-9 Mail</strong> bietet Ende-zu-Ende-Verschlüsselung für E-Mails. Ein experimentelles Projekt ist <a href='https://dev.guardianproject.info/projects/gpgandroid/wiki'>GnuPG für Android</a> von dem Guardian Project.</p><p>Read the <a href='https://emailselfdefense.fsf.org/'>Email Self-Defense</a> guide by the Free Software Foundation to learn how to encrypt your email messages.</p>",
```
Comment (strugee) [inline source/locales/de.json]: I assume these don't exist in other languages?

Comment (dschuermann) [inline source/locales/de.json]: They exist but I don't speak these languages. What you be done with them?

Comment (strugee) [inline source/locales/de.json]: @dschuermann ummm, not sure. I think probably the best course of action is to remove the outdated translations (which will cause them to fallback to English). /cc @nylira

**Comment by dschuermann** [comment]
@strugee done

**Comment by dschuermann** [comment]
@nylira Can you update the live site?

**Pull request merged**